src/site/content/en/handbook/audience/index.md

```diff
@@ -7,17 +7,18 @@ date: 2019-06-26
 description: |
   Definitions of the the key personas for web.dev and advice about how to write for them.
```

Comment:
I noticed that the descriptions are inconsistently written. Should it be a complete sentence, description (like alt text), or actionable (Learn how to write effective docs!). Is there guidance for this? There probably should be, since this is in every doc, right?

Reply:
The description field is going into the <meta> description field and will show up in search results. My hunch is we want it to be complete sentences.

Reply:
"A meta description tag should generally inform and interest users with a short, relevant summary of what a particular page is about. They are like a pitch that convince the user that the page is exactly what they're looking for. There's no limit on how long a meta description can be, but the search result snippets are truncated as needed, typically to fit the device width."

More info here: https://support.google.com/webmasters/answer/35624

Reply:
My impulse here is to provide guidelines (probably more or less the bit Lizzi quoted) plus a few templates people can choose from (e.g., "Learn to do x with y"). Also, seems to me this could be either a sentence or a noun phrase? But I would happily defer to others' expertise :)

Reply:
I think brevity (read 'pithy') is best here. Noun phrases should at least be allowed and that they are allowed should be communicated.

Reply:
Added note in Google Doc to incorporate this guidance.